### PR TITLE
albumartist_website: Handle multiple album artists

### PR DESCRIPTION
**Pull request by phw**
If there are multiple albumartists add the URLs for all of them. See https://tickets.metabrainz.org/browse/PICARD-2268

### CODE REVIEW

**Comment by phw** [comment]
@Sophist-UK Can you look over this?

File: plugins/albumartist_website/albumartist_website.py [hunk 1]
```diff
@@ -93,18 +94,22 @@ def website_process(self, artistId, response, reply, error):
...
             self.album_remove_request(album)
 
+    def add_websites_to_track(self, track, urls):
+        tm = track.metadata
+        websites = tm.getall('website')
+        websites += urls
+        websites.sort()
+        tm['website'] = websites
+        for file in track.iterfiles(True):
+            websites.append('foo %s' % file)
```
Comment (Sophist-UK) [inline plugins/albumartist_website/albumartist_website.py]: Eh? A statement for debugging?

Of course [sic.] I have ***never*** done anything remotely like this myself. 😉

**Review by Sophist-UK — COMMENTED** [review]
Aside from the one comment, LGTM as a desk review. I have not tested this personally.

File: plugins/albumartist_website/albumartist_website.py (hunk 1 above)
Comment (phw) [reply plugins/albumartist_website/albumartist_website.py]: Yes, debug code got in :( Fixed

**Comment by Sophist-UK** [comment]
Not sure why there was a force-push, but LGTM by code review.

**Review by Sophist-UK — APPROVED** [review]

**Pull request merged**